Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1683,10 +1683,7 @@
             F_OFD_SETLKWTIMEOUT))
 
 #if HAVE(SANDBOX_STATE_FLAGS)
-    (with-filter (require-not (state-flag "AppCacheDisabled"))
-        (allow system-fcntl (appcache-fcntl-commands)))
-    (with-filter (state-flag "AppCacheDisabled")
-        (deny system-fcntl (appcache-fcntl-commands)))
+    (deny system-fcntl (appcache-fcntl-commands))
 #else
     (allow system-fcntl (appcache-fcntl-commands))
 #endif
```

Comment:
If we are disabling AppCache on all OS versions, I think this could be removed along with `appcache-fcntl-commands`.

Reply:
I prefer doing this as a follow-up since this could potentially touch macOS downlevels.